File: 1-TwoSum/1-TwoSum.js
// Last updated: 3/6/2026, 2:09:46 PM
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    for(let i=0;i<nums.length;i++){
        for(let j=i+1;j<nums.length;j++){
            if(nums[i] + nums[j] == target){
               return [i,j]
            }
        }
    }
}
console.log(twoSum([2,7,11,15] , 9));

File: 1-TwoSum/1-TwoSum.cpp
// Last updated: 3/17/2026, 5:26:51 PM
1class Solution {
2public:
3    vector<int> twoSum(vector<int>& nums, int target) {
4        for(int i = 0; i < nums.size(); i++){
5            for(int j = i + 1; j < nums.size(); j++){
6                if(nums[i] + nums[j] == target){
7                    return {i, j};  
8                }
9            }
10        }
11        return {};
12    }
13};